src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java

```diff
@@ -58,12 +61,23 @@ public PartnerCenterResponse setLinks(ResourceLinks links) {
 	}
 
 	public String getContinuationToken() {
-		return continuationToken;
+		return extractContinuationToken().orElse(null);
```

Comment:
The old field was used for the fetch customer list. We would need to confirm the changes that you are proposing would not break the sync functionality